ServiceDocumentSampleComponents/force-app/main/default/lwc/mediumHeaderText/mediumHeaderText.html

```diff
@@ -0,0 +1,3 @@
+<template>
+  <h2 class="slds-text-heading_medium" style="{inlineStyle}">{text}</h2>
```

Comment:
You don't want the quotes around `{inlineStyle}` here.

Reply:
I think this was the only remaining one you did't already fix.

Reply:
Hey @khawkins, sorry for this missed fix. I just update the commit. Could you take a look again, thank you!!